### ISSUE TITLE
Don't crash if @irc-bridge user is mentioned on Slack

### ISSUE THREAD

**Issue by jvperrin** [ocf/slackbridge#51]
If the `@irc-bridge` user is mentioned on Slack, it crashes the bridge:

```
{'client_msg_id': '36240c5a-873a-431e-b336-3ee0a82385e1', 'user': 'UAEJSRH2R', 'text': '<@U1B6FUML3> <@U72GH8C0N> make sure you do dual key c:'}
Posting message to IRC
User bot posting message to IRC
Unhandled error in Deferred:
Unhandled Error
Traceback (most recent call last):
  File "/opt/slackbridge/venv/lib/python3.5/site-packages/twisted/internet/base.py", line 1261, in run
    self.mainLoop()
  File "/opt/slackbridge/venv/lib/python3.5/site-packages/twisted/internet/base.py", line 1270, in mainLoop
    self.runUntilCurrent()
  File "/opt/slackbridge/venv/lib/python3.5/site-packages/twisted/internet/base.py", line 896, in runUntilCurrent
    call.func(*call.args, **call.kw)
  File "/opt/slackbridge/venv/lib/python3.5/site-packages/twisted/internet/task.py", line 239, in __call__
    d = defer.maybeDeferred(self.f, *self.a, **self.kw)
--- <exception caught here> ---
  File "/opt/slackbridge/venv/lib/python3.5/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
    result = f(*args, **kw)
  File "/opt/slackbridge/slackbridge/bots.py", line 100, in empty_queue
    message.resolve()
  File "/opt/slackbridge/slackbridge/messages.py", line 83, in resolve
    self._post_to_irc(channel_name, user_bot)
  File "/opt/slackbridge/slackbridge/messages.py", line 161, in _post_to_irc
    self.raw_message['text'],
  File "/opt/slackbridge/slackbridge/bots.py", line 188, in post_to_irc
    IRCBot.channels,
  File "/opt/slackbridge/slackbridge/utils.py", line 163, in format_irc_message
    text = re.sub(r'<\@(U\w+)\|?(\w+)?>', user_replace, text)
  File "/opt/slackbridge/venv/lib/python3.5/re.py", line 182, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/opt/slackbridge/slackbridge/utils.py", line 127, in user_replace
    return readable or users[user_id].nickname
builtins.KeyError: 'U1B6FUML3'
```

**Comment by kkuehler** [comment]
Probably [this line](https://github.com/ocf/slackbridge/blob/fcb8f85d082aaf58d97fcd964872e0ebbd470bc1/slackbridge/main.py#L77). Ill take a look after work

**Comment by jvperrin** [comment]
I think adding them to the list before adding a new bot would be fine, my main concern there is that the bridge will try to send through messages coming from those bots when they should be filtered somehow. Ideally I think the behavior we want here is that if someone mentions `@irc-bridge` in Slack we want it to show up as `@irc-bridge` in IRC, and if we get messages from bot users on Slack then those should just show up through the slack-bridge IRC user, since we can't make a IRC bot for every possible Slack bot we might have (and they can change display name too like the bridge does).

**Comment by ja5087** [comment]
fixed with #58
